.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU:_
+
+## WHAT YOU DID
+_A brief description of the pull request:_ 
+
```

Comment:
Should we have a discussion points section? IE things the merger(?) wants to alert the team about and document for future generations in the PR.